data/data/bootstrap/files/usr/local/bin/bootstrap-pivot.sh.template

```diff
@@ -0,0 +1,42 @@
+{{if .IsOKD -}}
+#!/usr/bin/env bash
+set -euo pipefail
+# Rebase to OKD's OSTree container image.
+# This is required in OKD as the node is first provisioned with plain Fedora CoreOS.
+
+# shellcheck disable=SC1091
+. /usr/local/bin/bootstrap-service-record.sh
+. /usr/local/bin/release-image.sh
+
+# Pivot bootstrap node to OKD's OSTree image
+if [ ! -f /opt/openshift/.pivot-done ]; then
+{{if .IsFCOS -}}
+  MACHINE_OS_IMAGE=$(image_for fedora-coreos)
+{{else if .IsSCOS -}}
+  MACHINE_OS_IMAGE=$(image_for centos-stream-coreos-9)
+{{end -}}
+  echo "Pulling ${MACHINE_OS_IMAGE}..."
+  while true
+  do
+    record_service_stage_start "pull-okd-os-image"
+    if podman pull --quiet "${MACHINE_OS_IMAGE}"
+    then
+        record_service_stage_success
+        break
+    else
+        record_service_stage_failure
+        echo "Pull failed. Retrying ${MACHINE_OS_IMAGE}..."
+    fi
+  done
+
+  record_service_stage_start "rebase-to-okd-os-image"
+  mnt="$(podman image mount "${MACHINE_OS_IMAGE}")"
```

Comment:
Looking at this, it seems like we're going through a lot of *extra* complexity to pull the code added in https://github.com/openshift/os/pull/990 and it'd be way simpler to just include that code here too.

Reply:
OK, understood.  How often does it change though?  

(One argument for not having it in the OS repository is that we don't actually test cluster installs there yet (embarrassingly enough, it shouldn't be hard to fix but it's kind of pointless because 95% of the changes that could break cluster installs are RPM changes, which *aren't* gated on PRs))

We could in theory of course move some of this logic to a separate (privileged) container image that we pull and run, but I'm not sure which github repo that'd be, and it'd have the same problem that it would really become heavily load-bearing and need e2e tests there too.
(In theory for example some of this logic could live in Go code in the MCO binary too, dunno if that's better or not)

Reply:
We did have a couple of changes there from time to time, especially recently with the migration to the ostree container rebase, and from time to time also regarding the OKD-specific manifests shipped there.

For now this PR doesn't change those mechanics in OKD/FCOS, but going forward I think doing this directly here in the installer would be OK.

I definitely want to loop in @vrutkovs on this.

Reply:
>How often does it change though?

Rarely.

>(In theory for example some of this logic could live in Go code in the MCO binary too, dunno if that's better or not)

I think it would be preferable. We already have okd-specific build tag there and the team is actively working on OKD/SCOC